pds_doi_service/core/util/initialize_production_deployment.py

```diff
@@ -1,366 +1,388 @@
+#               -X GET -H "Content-Type: application/xml" -H "Accept: application/xml > my_input.xml
 #       Then the file my_input.xml can be edited to include the lidvid in the accession_number tag
 #           <accession_number>a:b:c::1.0</accession_number>
-# Note: As of 10/05/2020, there are 105 records that does not have info for lidvid.  Ron has been notified.
-# Note: As of 10/06/2020, there are 1058 DOIs in the OPS OSTI server associated with the NASA-PDS account.
+# Note: As of 10/05/2020, there are 105 records that does not have info for lidvid.
+#       Ron has been notified.
+# Note: As of 10/06/2020, there are 1058 DOIs in the OPS OSTI server associated
+#       with the NASA-PDS account.
 
 import argparse
 import logging
 import os
-
 from datetime import datetime
 
-from pds_doi_service.core.db.doi_database import DOIDataBase
-from pds_doi_service.core.input.exceptions import InputFormatException, CriticalDOIException
-from pds_doi_service.core.util.config_parser import DOIConfigUtil
-from pds_doi_service.core.util.general_util import get_logger
-
+from pds_doi_service.core.input.exceptions import (InputFormatException,
+                                                   CriticalDOIException)
 from pds_doi_service.core.outputs.osti_web_client import DOIOstiWebClient
 from pds_doi_service.core.outputs.osti_web_parser import DOIOstiWebParser
-
 from pds_doi_service.core.outputs.transaction_builder import TransactionBuilder
-from pds_doi_service.core.outputs.transaction import Transaction
-
+from pds_doi_service.core.util.config_parser import DOIConfigUtil
+from pds_doi_service.core.util.general_util import get_logger
 
 # Get the common logger and set the level for this file.
-logger = get_logger('pds_doi_core.util.initialize_production_deployment')
+logger = get_logger(__name__)
 logger.setLevel(logging.INFO)
 
 m_doi_config_util = DOIConfigUtil()
 m_config = m_doi_config_util.get_config()
 
+
 def create_cmd_parser():
     parser = argparse.ArgumentParser(
-        description='Script to import existing DOIs into local database\n')
-    parser.add_argument("-i", "--input", help="Input file to import existing DOIs from or URL of OSTI server.",
-                        required=False)
-    parser.add_argument("-s", "--submitter_email", help="The email address of the user performing the initialization of deployment database.",
-                        required=True)
-    parser.add_argument("-d","--db_name", help="Optional database name instead of one from config.",
-                        required=False)
-    parser.add_argument("--dry-run", help="Flag to suppress the writing of DOIs to database.",
-                        required=False,action="store_true")
-    parser.add_argument("--debug", help="Flag to print debug statements.",
-                        required=False,action="store_true")
+        description='Script to import existing DOIs from OSTI into the local'
```

Comment:
Personally, I think the longer statement lines are far more readable than blind adherence to PEP8's "80 columns" rule but 🤷‍♀️